src/domain/chainContext.ts

```diff
@@ -24,6 +31,34 @@ const WATCHDOG_FREQUENCY = 5 * 1000; // 5 seconds
 
 const MULTICALL3 = "0xcA11bde05977b3631167028862bE2a173976CA11";
 
+enum ChainSync {
+  /** The chain is currently in the warm-up phase, synchronising from contract genesis or lastBlockProcessed */
+  SYNCING = "SYNCING",
+  /** The chain is in sync with the latest block */
+  IN_SYNC = "IN_SYNC",
+  /** The chain is in an unknown state based on duration of time since a block was processed */
+  UNKNOWN = "UNKNOWN",
```

Comment:
If this models that the last block you sold is old. Why not calling it `OUT_OF_SYNC` or `STALE` or similar?nit:

Reply:
`STALE` sounds good. Done.